index.bs

```diff
@@ -268,6 +268,54 @@ https://school.register.gov.uk/proof/entry/123/merkle:sha-256
 </pre>
 </div>
 
+## Consistency proof resource ## {#consistency-proof-resource}
+<dl class="resource"><dt>Path</dt><dd>/proof/consistency/{total-entries-1}/{total-entries-2}/{proof-identifier}</dd></dl>
+
+A consistency proof proves the append-only property of a register.
+
+Assuming n is the greatest of total-entries-1 and total-entries-2 and m is the smallest of total-entries-1 and total-entries-2 (m <= n), a consistency proof for a register of n entries and a previous version of the same register with m entries is the information required to prove that the first m entries are equal for both, given a [[#register-proof-resource]] for both versions of the register.
```

Comment:
I think we should require that `total-entries-1` is smaller than `total-entries-2`, rather than putting extra logic into the register to find the min and max.